src/ruby/ext/grpc/rb_grpc.h

```diff
@@ -85,4 +85,9 @@ void grpc_ruby_init();
     abort();                                                      \
   }
 
+#ifndef RB_NOGVL_OFFLOAD_SAFE
```

Comment:
is this meant to be toggled by users when compiling the gem?

If not, we may just hardcode to zero